ddp-workspace/projects/ddp-sdk/src/lib/models/googleAnalyticsEvent.ts

```diff
@@ -0,0 +1,12 @@
+import { GTagEvent } from './gtagEvent';
+import { AnalyticsEvent } from './analyticsEvent';
+
+export type GoogleAnalyticsEvent = AnalyticsEvent | GTagEvent;
+
+export function isGtagEvent(event: GoogleAnalyticsEvent): event is GTagEvent {
+    return 'event_name' in event && 'parameters' in event;
+}
+
+export function isAnalyticsEvent(event: GoogleAnalyticsEvent): event is AnalyticsEvent {
+    return 'hitType' in event;
+}
```

Comment:
Added GoogleAnalytics common type for backwards compatibility from gtag.js. All our other studies at this point use ga.js .